irm/google/cloud/irm_v1alpha2/gapic/transports/incident_service_grpc_transport.py

```diff
@@ -0,0 +1,553 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import google.api_core.grpc_helpers
+
+from google.cloud.irm_v1alpha2.proto import incidents_service_pb2_grpc
+
+
+class IncidentServiceGrpcTransport(object):
+    """gRPC transport class providing stubs for
+    google.cloud.irm.v1alpha2 IncidentService API.
+
+    The transport provides access to the raw gRPC stubs,
+    which can be used to take advantage of advanced
+    features of gRPC.
+    """
+
+    # The scopes needed to make gRPC calls to all of the methods defined
+    # in this service.
+    _OAUTH_SCOPES = ()
```

Comment:
@jdpedrie pointed out that the GAPIC config for IRM is missing the scoping information needed here.

Reply:
@busunkim96 The GAPIC config for IRM has been [updated to include scopes](https://github.com/googleapis/googleapis/commit/84f9bf2c274694a632b67ca0efcad20c56ead8cd).

Reply:
Thank you, I'll re-generate.